kolibri/plugins/management/assets/src/vue/class-enroll-page/index.vue

```diff
@@ -353,5 +354,63 @@
 
   .col-name, .col-username
     width: 45%
+    
+  .results-text
+    font-size: 0.9375rem
+    
+  .toolbar
+    margin-top: 30px
+    
+  .toolbar:after
+    content: ''
+    display: table
+    clear: both
+    
+  input[type='search']
```

Comment:
the second option might be more trouble than it's worth unless the code is shared verbatim.

Yes, commenting on where the code was copied from would be helpful